src/main/java/io/cdap/e2e/pages/actions/CdfConnectionActions.java

```diff
@@ -382,10 +383,8 @@ public static void clickOnAddEntityButton() {
   }
 
   public static void fixAllIfPrompted() {
-    if (SeleniumDriver.getDriver().findElements(By.xpath(CdfConnectionLocators.fixAllButtonXpath)).size() != 0) {
-      WaitHelper.waitForElementToBeDisplayed(CdfConnectionLocators.fixAllButtonElement());
-      WebElement fixAllButton = (SeleniumDriver.getDriver().findElement(CdfConnectionLocators.locatorOfFixAll()));
-      fixAllButton.click();
+    clickIfDisplayed(CdfConnectionLocators.locatorOfFixAll()); {
+      clickIfDisplayed(CdfConnectionLocators.locatorOfFixAll(), ConstantsUtil.SMALL_TIMEOUT_SECONDS);
```

Comment:
Why do we need to call the same method again?

Reply:
resolved